import_and_save.py

```diff
@@ -0,0 +1,85 @@
+import click
+import csv
+from importlib import import_module
+
+def save_into_mysql(filename, username, password):
+    try:
+        module = import_module('MySQLdb')
+        db_name = 'cli_test'
+        mydb_conn = module.connect(user=username, passwd=password)
+        cur = mydb_conn.cursor()
+        cur.execute("""CREATE DATABASE IF NOT EXISTS """ + db_name)
+        mydb_conn = module.connect(user=username, passwd=password, db=db_name)
+        cur = mydb_conn.cursor()
+        csv_data = csv.reader(file(filename))
+        lst = ['Name', 'Age', 'Address']
+        table_name = "People"
+        createsqltable = """CREATE TABLE IF NOT EXISTS """ + table_name + " (" + " VARCHAR(250),".join(lst) + " VARCHAR(250))"
+        cur.execute(createsqltable)
+        for row in csv_data:
+            cur.execute('INSERT INTO People(Name,Age,Address)''VALUES(%s, %s, %s)',row)
+        mydb_conn.commit()
+        cur.close()
+    except:
+        print('Sorry Something went wrong')
+
+def save_into_sqlite(filename):
+    try:
+        import sqlite3
+        con = sqlite3.connect("my_test.db")
+        cur = con.cursor()
+        cur.execute("CREATE TABLE IF NOT EXISTS t(Name, Age, Address);") # use your column names here
+        with open(filename,'rb') as fin: # `with` statement available in 2.5+
+        # csv.DictReader uses first line in file for column headings by default
+            dr = csv.DictReader(fin) # comma is default delimiter
+            to_db = [(i['Name'], i['Age'], i['Address']) for i in dr]
+        cur.executemany("INSERT INTO t (Name, Age, Address) VALUES (?, ?, ?);", to_db)
+        con.commit()
+        con.close()
+    except:
+        print('Sorry Something went wrong')
+
+def save_into_postgres(filename, username, password):
+    try:
+        module = import_module('psycopg2')
+        from psycopg2.extensions import ISOLATION_LEVEL_AUTOCOMMIT
```

Comment:
imports should be defined at top of the file, please use pep8 style guideline.

Reply:
I didn't want every time all the modules to import thats why did inside the method